### PR TITLE
Wrote data to a file

### PR DESCRIPTION
**Pull request by abhissssss**

### CODE REVIEW

File: src/main/java/com/abhisheksingh/simplegame/location/Locations.java [hunk 1]
```diff
@@ -1,14 +1,40 @@
...
+                if (locFile!=null){
+                    locFile.close();
+                }
+            }
+            catch (IOException e){
+                  e.printStackTrace();
+            }
+        }
+    }
     static {
```
Comment (shankhadeep-ghoshal) [inline src/main/java/com/abhisheksingh/simplegame/location/Locations.java:38]: What is the use of the static block?

File: src/main/java/com/abhisheksingh/simplegame/location/Locations.java [hunk 2]
```diff
@@ -1,14 +1,40 @@
...
 import java.util.Map;
 import java.util.Set;
 
 public class Locations implements Map<Integer,Location> {
-    private static Map <Integer,Location> locations = new HashMap<Integer,Location>();
-
-
+    private static Map <Integer,Location> locations = new HashMap<>();
+
+    public static void main(String[] args) {
```
Comment (shankhadeep-ghoshal) [inline src/main/java/com/abhisheksingh/simplegame/location/Locations.java:13]: Break down this entire blob of code into smaller chunks and encapsulate them separately

**Review by shankhadeep-ghoshal — CHANGES_REQUESTED** [review]
Major changes requested

File: src/main/java/com/abhisheksingh/simplegame/location/Locations.java (hunk 1 above)
Comment (abhissssss) [reply src/main/java/com/abhisheksingh/simplegame/location/Locations.java:38]: because I want the code in this block to be loaded before main

**Pull request merged**